generic-examples/amqp/README.md

```diff
@@ -1,5 +1,46 @@
 # AMQP Camel K examples
 
-Find useful examples about how to use AMQP in a Camel K integration.
+These examples demonstrate how to use AMQP in a Camel K integration.
 
-You may find useful instructions [how to install a JMS/AMQP Broker on Kubernetes](./artemis/) for demo purposes.
\ No newline at end of file
+You can find more information about Apache Camel and Apache Camel K on the [official Camel website](https://camel.apache.org).
+
+## Before you begin
+
+Read the general instructions in the [root README.md file](/README.md) for setting up your environment and the Kubernetes cluster before looking at this example.
+
+Make sure you've read the [installation instructions](https://camel.apache.org/camel-k/latest/installation/installation.html) for your specific
+cluster before starting the example.
+
+## Additional Requirements for running this example
+
+**An AMQP Broker**: needed MoM for running the examples. For installation instructions, see [how to install a JMS/AMQP Broker on Kubernetes](./artemis/) for demo purposes.
+
+## Understanding the Example
+[`AmqpConnectionBeanProducer.java`](./AmqpConnectionBeanProducer.java) defines a route that generates a message every second and sends it to an amqp queue. \
```

Comment:
The README is already well written, but if I could add anything to it this section could be made bullet points instead of a single paragraph and it could become more readable.

Reply:
Done